setup.py

```diff
@@ -46,7 +46,7 @@ def get_extras_require():
         'checking': ['autopep8', 'hacking'],
         'testing': [
             'pytest', 'mock', 'bokeh', 'plotly', 'chainer>=5.0.0', 'xgboost', 'mpi4py', 'lightgbm',
-            'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma'
+            'catboost', 'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma'
```

Comment:
There are no unit tests that depend on on `catboost`, so we don't have to add it to the list at this time.
```suggestion
            'keras', 'mxnet', 'scikit-optimize', 'tensorflow', 'cma'
```